beets/ui/__init__.py

```diff
@@ -471,29 +477,23 @@ def get_replacements():
     return replacements
 
 
-def _pick_format(album, fmt=None):
-    """Pick a format string for printing Album or Item objects,
-    falling back to config options and defaults.
-    """
+def format_(obj, fmt):
+    """Print a object, intended for Album and Item
+
+    This is equivalent to `format`, but the spec can be None
+    `fmt` is mandatory for otherwise one can just call `format(my_object)`"""
```

Comment:
But on the other hand:

    >>> import beets.library
    >>> i = beets.library.Item()
    >>> format(i)
    u' -  - '

meaning that you can just leave off the argument. Are there places where we really need to use `None` transparently? Or would a call like `format(item, fmt or '')` work for places where `fmt` might be `None`?